.dockerignore

```diff
@@ -202,5 +202,9 @@ cli/src/semgrep/bin/*
 # Artifacts from CI
 **/scripts/release/release_body.txt
 
+# nix
+**/result
+# direnv
```

Comment:
what is direnv? If it's part of nix, then maybe can just have the # nix and remove # direnv.